frontend/packages/console-plugin-shared/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './components';
+export * from './utils';
```

Comment:
Can I suggest a package like `datetime` that has a more specific meaning instead of a generic `utils` package?

Reply:
👍